### DISCUSSION TITLE
Environment variables and their path to config

### DISCUSSION THREAD

**Discussion by useronkel** [drakkan/sftpgo#1450] [General]
Hi.
I'm getting stuck with environment in /etc/sftpgo/sftpgo.env.
I'm trying to set the version appearing in the footer of the base.html template in webadmin/
```
            {{if .LoggedAdmin.Username}}
            <!-- Footer -->
            <footer class="sticky-footer bg-white">
                <div class="container my-auto">
                    <div class="copyright text-center my-auto">
                        <span>SFTPGo {{.Version}}</span>
                    </div>
                </div>
            </footer>
            <!-- End of Footer -->
            {{end}}
```
The interesting part is the {{.Version}}, but I do not get the right path to this variable.
My suggestion is SFTPGO_HTTPD__BINDINGS__0__BRANDING_WEB_ADMIN_VERSION, but that won't work ;-)

Do I misunderstand the path or is there no configurable version in the config for the footer?

**Comment by drakkan** [comment]
Changing the version is not supported

**Reply by useronkel** [reply]
Thx for clearing this

Edith says: Is it supported in the Pro or Enterprise plan?

**Comment by drakkan** [comment]
> Thx for clearing this
> 
> Edith says: Is it supported in the Pro or Enterprise plan?

Please note that in SFTPGo a feature is not added just because it is requested by a paying company, it is not proprietary software, making money is not the priority. New features should add value to the software and be useful to many users, not solve very specific needs of a single company.

If you want to support the project just to have a feature avoid it, thanks.
